src/Platform/Microsoft.Testing.Platform/OutputDevice/TerminalOutputDevice.cs

```diff
@@ -98,6 +98,13 @@ public TerminalOutputDevice(ITestApplicationCancellationTokenSource testApplicat
         bool noAnsi = parseResult.IsOptionSet(TerminalTestReporterCommandLineOptionsProvider.NoAnsiOption);
         bool noProgress = parseResult.IsOptionSet(TerminalTestReporterCommandLineOptionsProvider.NoProgressOption);
 
+        bool showPassed = false;
+        OptionRecord? output = parseResult.Options.FirstOrDefault(o => o.Option == TerminalTestReporterCommandLineOptionsProvider.OutputOption);
+        if (output != null && output.Arguments.Length > 0 && "Detailed".Equals(output.Arguments[0], StringComparison.OrdinalIgnoreCase))
```

Comment:
Could we use some constants or some enum instead of duplicating the string?

Reply:
used const.